test/csuite/wt11634_get_set_key_value/main.c

```diff
@@ -0,0 +1,154 @@
+/*-
```

Comment:
Hi Ravi, do we need this new test file? I'd consider performing similar tests using the Catch2 framework instead, and adding them to those you have added in test_cursor_set_raw_key_value.cpp.